administration/src/cards/Card.test.ts

```diff
@@ -139,6 +140,28 @@ describe('Card', () => {
     })
   })
 
+  it.each(['$tefan Mayer', 'Karla K.', 'Karla 1234 '])(
```

Comment:
When i enter a name with a space at the end i get an error message. I think this should just be valid but deleted internally

Reply:
since we can not trim it from the state i added this to the generateCardInfo and to the formField check.
Good find since mobile device auto completion would make this annoying